slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md

```diff
@@ -31,23 +31,41 @@ The following method allows you to create a PowerPoint presentation from a PDF d
 | :- | :- | :- | :- | :- |
 |name|string|path|true|The file name of the new presentation or existing one.|
 |pdf|file|formData|true|The input data of a PDF document.|
+|options|[PdfImportOptions](#import-options)|header|false|The password to open the presentation.|
```

Comment:
There is an incorrect comment for the `options` parameter here.

Reply:
will be fixed